### ISSUE TITLE
Change spaces in snippets to tabs

### ISSUE THREAD

**Issue by Hadoukanen** [vuetifyjs/vuetify-vscode#5]
I would suggest that you use tabs inside the snippets instead of spaces in vuetify.json.
Like this.

```
"Vuetify default layout": {
    "prefix": "vPreDefinedLayoutDefault",
    "body": [
      "<v-app>",
      "\t<v-navigation-drawer app></v-navigation-drawer>",
      "\t<v-toolbar app></v-toolbar>",
      "\t<v-content>",
      "\t\t<v-container fluid>",
      "\t\t\t<router-view></router-view>",
      "\t\t</v-container>",
      "\t</v-content>",
      "\t<v-footer app></v-footer>",
      "</v-app>"
    ],
    "description": "Vuetify default layout"
  },
```
Visual studio code changes tabs to spaces if that is detected / set in the html / vue file,
It seems that Visual Studio code can't change spaces to tabs that easily. might have to do with how many tabs spacers there are like 2 or 4. but if you use tabs it does not matter and will be converted to what ever you have set or what is detected.

**Comment by abhaywawale** [comment]
Will be done in next release.
Thanks for feedback.

**Comment by abhaywawale** [comment]
Released v0.2.0 which will solve the problem with indent. Please feel free to open another issue if problem persists.
